### PR TITLE
Added SmallRye areas in autoissuelabeler.yml

### PR DESCRIPTION
**Pull request by phillip-kruger**
New labels needed 
- area/tracing
- area/metrics
- area/health
- area/fault-tolerance

### CODE REVIEW

File: .github/autoissuelabeler.yml [hunk 1]
```diff
@@ -47,12 +47,36 @@
...
-  notify: [phillip-kruger,jmartisk]  
+  notify: [phillip-kruger,jmartisk]
+- labels: [area/reactive-messaging, area/smallrye]
+  title: "(?i).*reactive.messaging.*"
+  notify: [kenfinnigan,cescoffier,phillip-kruger]
+- labels: [area/tracing, area/smallrye]
+  title: "(?i).*(trace|tracing|telemetry).*"
+  notify: [kenfinnigan,Ladicek]
+- labels: [area/config, area/smallrye]
+  title: "(?i).*config.*"
```
Comment (gsmet) [inline .github/autoissuelabeler.yml:63]: IMHO, this one is going to be a pain but you can try :).

File: .github/autoissuelabeler.yml [hunk 2]
```diff
@@ -47,12 +47,36 @@
...
   title: "(?i).*openapi.*"
   notify: [EricWittmann,MikeEdgar,phillip-kruger]
-- labels: [area/graphql]
+- labels: [area/graphql, area/smallrye]
   title: "(?i).*graphql.*"
-  notify: [phillip-kruger,jmartisk]  
+  notify: [phillip-kruger,jmartisk]
+- labels: [area/reactive-messaging, area/smallrye]
+  title: "(?i).*reactive.messaging.*"
+  notify: [kenfinnigan,cescoffier,phillip-kruger]
```
Comment (gsmet) [inline .github/autoissuelabeler.yml:58]: I think Ken just added that one in another PR, could you rebase and check? Talking about that one: https://github.com/quarkusio/quarkus/pull/11001 .

File: .github/autoissuelabeler.yml (hunk 1 above)
Comment (phillip-kruger) [reply .github/autoissuelabeler.yml:63]: You mean it will match too much ?

Comment (gsmet) [reply .github/autoissuelabeler.yml:63]: Yeah, that's what I'm worried about.

I even wonder if we don't have `config` in the default template. Oh yes, we do: https://github.com/quarkusio/quarkus/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.md&title= :)

**Comment by phillip-kruger** [comment]
Closing this one. Opened #11020

**Pull request closed without merging**